.github/workflows/add-to-projects.yml

```diff
@@ -99,10 +99,17 @@ jobs:
           github-token: ${{ secrets.GH_PROJECT_AUTOMATION_TOKEN }}
           # any of the following labels:
           labeled: component/clients, component/spring-sdk, component/camunda-process-test, component/c8-api
-      - id: add-to-qualityboard
-        name: Add to Quality Board project
-        uses: actions/add-to-project@v1.0.2
+      - id: add-bug-to-quality-board
```

Comment:
Can you make a test in another (small, test) repo to see that it works indeed? Feel free to share the link if you already had.

Reply:
LGTM, thanks.